forge/forge/config/config.py

```diff
@@ -210,7 +211,7 @@ def build_config_from_env(cls, project_root: Path = PROJECT_ROOT) -> Config:
 
 def assert_config_has_openai_api_key(config: Config) -> None:
     """Check if the OpenAI API key is set in config.py or as an environment variable."""
-    key_pattern = r"^sk-(proj-)?\w{48}"
+    key_pattern = r"^sk-[\w-]{40,}"
```

Comment:
If the only changed variable component is that `proj` can also be any `\w+`, should the regex be:
```suggestion
    key_pattern = r"^sk-(\w+-)?\w{48}"
```

Reply:
It can be any number of dashes for an indeterminate length. There’s not a “valid” regex for it so the suggestions is to be loose with it

Reply:
but then the key part is still gonna be 48 chars right?

Reply:
There was nowhere that said it would be, and mine was len = 63 including `sk-` for the one I generated

Reply:
```suggestion
    key_pattern = r"^sk-[\w-]*\w{48}"
```